R/modal.R

```diff
@@ -113,6 +114,26 @@
 #'   })
 #' }
 #'
+#' if (interactive()) {
+#' library(shiny)
+#' library(shiny.semantic)
+#' shinyApp(
+#'   ui = semanticPage(
+#'     actionButton("show", "Show modal dialog")
+#'   ),
+#'   server = function(input, output) {
+#'     observeEvent(input$show, {
+#'       showModal(modalDialog(
+#'         title = "Important message",
+#'         "This is an important message!", easyClose = FALSE
+#'       ))
+#'       removeModal()
```

Comment:
I am not sure if this example works as intended. When I run it, I see the button, I click it, but there is no modal (nothing happens). But when I click multiple times, I can spot that something is displayed, but then immediately removed. Maybe `removeModal()` should be run after 3 seconds?